### PR TITLE
Add support for execution metrics to GenericSQL integration.

### PR DESCRIPTION
**Pull request by mcvic1rj**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] Ready

## Related Issues
No related issues

## Description
Added support for Execution Metrics to track failures in the GenericSQL Integration.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/GenericSQL/ReleaseNotes/1_3_4.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Generic SQL
+
+- Added execution metric reporting to GenericSQL Calls.
```
Comment (yaakovpraisler) [inline Packs/GenericSQL/ReleaseNotes/1_3_4.md:6]:
```suggestion
- Added support for API Execution Metric reporting to GenericSQL Calls.
```

File: Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py [hunk 2]
```diff
@@ -652,7 +675,19 @@ def main():
...
+                                    entry_type=entryTypes["note"],
+                                    outputs=outputs,
+                                    raw_response=raw,
+                                    ignore_auto_extract=False,
+                                    content_format=formats["text"] if isinstance(raw, STRING_TYPES) else formats['json']
+                                    )
+
+            command_results.append(result)
+            return_results(command_results)
+
```
Comment (yaakovpraisler) [inline Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:690]: Do you need this for support with the execution metrics? if not, maybe it's worth not to change it to not break something mistakenly

**Review by yaakovpraisler — COMMENTED** [review]
Hey @mcvic1rj, Thank you for your contribution! It looks good, just a few comments -

Also, Let's have a demo for this.

File: Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py (hunk 2 above)
Comment (mcvic1rj) [reply Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py]: I reverted this block back to the deprecated command.

**Comment by mcvic1rj** [comment]
I will work on grabbing a few screenshots of this integration working.

**Comment by mcvic1rj** [comment]
Changes to the integration support tracking of the metrics as seen below: 
image

Currently the metrics are broken into 3 sections, `Success`, `General Error`, and `Connection Error`.

**Comment by yaakovpraisler** [comment]
Hey @mcvic1rj, Please share your available times for a quick Zoom call for a demo.

**Review by yaakovpraisler — APPROVED** [review]

**Comment by yaakovpraisler** [comment]
@mcvic1rj Thanks for the demo, good work!

**Pull request merged**